html/pfappserver/root/static.alt/src/styles/_input-group.scss

```diff
@@ -1,11 +1,4 @@
 @import "../../node_modules/bootstrap/scss/input-group";
 
 .input-group {
-    // Create a stacking context so the pf-form-chosen (vue-multiselect)
-    // component is displayed above any b-input-group component.
-    z-index: 1;
-
-    .form-group.is-focus & {
-        z-index: 2;
-    }
-}
```

Comment:
This fix is no longer necessary?

Reply:
This fix was causing the pfFormChosen drop-down container to appear below other elements both either before or after this element in the DOM. I removed it to correct the issue it was causing.

I forgot it's original purpose, so I'm unable to tell if it's still needed, or not.

Reply:
This is my fix. Let's merge and I'll see if it's still necessary.